homeassistant/components/prosegur/alarm_control_panel.py

```diff
@@ -67,17 +69,21 @@ async def async_update(self):
             self._attr_available = False
             return
 
+        assert self._installation
```

Comment:
Instead of this, can we add a type hint to the class and not set `self._installation` in `__init__`?

Reply:
done